docs/src/modules/components/ThemeContext.js

```diff
@@ -245,13 +249,14 @@ export function ThemeProvider(props) {
   React.useEffect(() => {
     // Expose the theme as a global variable so people can play with it.
     window.theme = theme;
-    window.createTheme = createTheme;
+    window.createTheme = createMdTheme;
   }, [theme]);
 
+  // TODO: remove MdThemeProvider, top level layout should render the default theme.
```

Comment:
I think that this is really important, we want to show the default look in the docs.